Px.Utils/Validation/Enums.cs

```diff
@@ -50,6 +50,13 @@ public enum ValidationFeedbackRule
         KeywordIsNotInUpperCase = 22,
         KeywordExcessivelyLong = 23,
         IllegalCharactersInSpecifierPart = 24,
-        EntryWithMultipleValues = 25
+        EntryWithMultipleValues = 25,
```

Comment:
I added default values to the existing validation feedback rules. I'd suggest following the same convention with the new ones.